hackathon_site/dashboard/frontend/src/components/general/OrderTables/OrderTables.tsx

```diff
@@ -88,3 +102,206 @@ export const GeneralOrderTableTitle = ({
         )}
     </Container>
 );
+
+export const GeneralPendingTable = ({
+    pendingOrder,
+    extraColumn,
+}: {
+    pendingOrder: OrderInTable;
+    extraColumn?: {
+        header: ReactElement;
+        body: (id: number) => ReactElement;
+    };
+}) => {
+    const hardware = useSelector(hardwareSelectors.selectEntities);
+
+    return (
+        <>
+            <GeneralOrderTableTitle
+                orderId={pendingOrder.id}
+                orderStatus={pendingOrder.status}
+            />
+            <TableContainer component={Paper} elevation={2} square={true}>
+                <Table className={styles.table} size="small" aria-label="pending table">
+                    <TableHead>
+                        <TableRow>
+                            <TableCell className={styles.widthFixed} />
+                            <TableCell className={styles.width6} align="left">
```

Comment:
![image](https://user-images.githubusercontent.com/84813627/186009478-e2c7df74-7d8a-4a17-aec3-f3889d9deb7f.png)
I think the table formatting is a bit too differentbetween the ```Ready for pickup``` orders and the ```In progress``` orders, which becomes more apparent on bigger screens. The ```Name``` column is too far to the right for the ```In progress``` order.